src/test/java/org/opensearch/security/SlowIntegrationTests.java

```diff
@@ -163,6 +164,7 @@ public void testNodeClientDisallowedWithNonServerCertificate2() throws Exception
     }
 
     @Test
+    @Ignore
```

Comment:
Muting this flaky test. This test was previously muted on main and fixed, but the fix has not been backported to 1.3: https://github.com/opensearch-project/security/pull/3763